src/index.js

```diff
@@ -0,0 +1,23 @@
+import './style.css';
+import Todo from './modules/todo.js';
+import {
+  desc,
+  form,
+  refresh,
+} from './modules/element.js';
+
+const todo1 = new Todo();
+
+todo1.renderPage();
+
+desc.addEventListener('keypress', (e) => {
+  if (e.key === 'Enter' && desc.value !== '') {
+    todo1.addTodo();
+    e.preventDefault();
+    form.reset();
+  }
+});
```

Comment:
This milestone cover 
- removing items from the list
- editing the items in the list
- When items are removed, the index should be updated to match the total number of items remaining in the list.

You have just implemented the add functionality, kindly implement the other functionalities.

<img width="1048" alt="image" src="https://github.com/obibaadoma/To-Do-List/assets/27709832/560420f3-22d5-4aa5-9d00-e459c422f435">

Reply:
There more changes that are needed